### ISSUE TITLE
Alternatives

### ISSUE THREAD

**Issue by csi-adziahel** [jml/servant-template#26]
This template seems abandoned, any suggestions on what to use instead of it?

**Comment by jml** [comment]
Nope. I keep making similarish Haskell projects, but haven't figured out a good workflow for keeping this up-to-date.
